docs/contributions/unity-gdk-coding-standards.md

```diff
@@ -69,26 +67,22 @@ If you don't have ReSharper, or want to format from the command line, run `ci/li
 
 ## Tests
 
-* Name test fixtures as `<class>Test`
+* Name test fixtures as `<class>Tests`
 * Name test methods as `<method>_should_<action>_when_<conditions>`
 
 ## Unity specific
 
-* We are using the experimental support for `.NET 4.6` in the Unity GDK (mandated by use of the Unity ECS). This equates to `C# 6` so newer language features are supported.
```

Comment:
Could also specify that it's the non-experimental .NET 4.x support. Which is what Unity calls it. That is C# 7 ish though.